### ISSUE TITLE
Loss keep going up and down

### ISSUE THREAD

**Issue by AnakTeka** [google-research/bert#830]
![image](https://user-images.githubusercontent.com/3157407/63993841-81e06380-cb1c-11e9-815d-212c366e74fd.png)

this is my train command 
```
python run_pretraining.py 
--input_file="data/wikipedia_corpus/final_tfrecords_sharded/tf_examples.tfrecord000*"   
--output_dir=results/  
--do_train=True  
--do_eval=True  
--bert_config_file=data/bert_config.json    
--train_batch_size=32   
--max_seq_length=128   
--max_predictions_per_seq=20   
--num_train_steps=1000000   
--num_warmup_steps=40000 
--save_checkpoints_steps=5000   
--learning_rate=1e-5
```

any ideas?

**Comment by GokulNC** [comment]
@AnakTeka How do you get the loss plot?

**Comment by guotong1988** [comment]
Same question.
